sites/star.azure.zooniverse.org.conf

```diff
@@ -0,0 +1,24 @@
+server {
+    include /etc/nginx/ssl.default.conf;
+    server_name "~^(?P<subdomain>.*)\.azure\.zooniverse\.org$";
+
+    location / {
+        resolver 8.8.8.8;
```

Comment:
not blocking - are we happy using google dns servers here? Longer term should we use azure ones / cloudflare 1.1.1.1 (increased privacy)?

Reply:
OOS but opening an ADR or issue would be great.